ui/src/main/js/page/channel/azure/AzureBoardsPageLayout.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import PageHeader from 'common/component/navigation/PageHeader';
-import AzureBoardTale from 'page/channel/azure/AzureBoardTable';
+import AzureBoardTable from 'page/channel/azure/AzureBoardTable';
+import { useLocation } from 'react-router-dom';
 import { AZURE_BOARDS_INFO } from 'page/channel/azure/AzureBoardsModel';
 
-const AzureBoardsLayout = ({ readonly, allowDelete }) => (
-    <div>
-        <PageHeader
-            title={AZURE_BOARDS_INFO.label}
-            description="Configure the Azure Boards instance that Alert will send issue updates to."
-            icon={['fab', 'windows']}
-        />
-        <AzureBoardTale readonly={readonly} allowDelete={allowDelete} />
-    </div>
-);
+const AzureBoardsLayout = ({ readonly, allowDelete }) => {
+    const location = useLocation();
+
+    return (
+        <div>
+            <PageHeader
+                title={AZURE_BOARDS_INFO.label}
+                description="Configure the Azure Boards instance that Alert will send issue updates to."
+                icon={['fab', 'windows']}
+            />
+            <AzureBoardTable readonly={readonly} allowDelete={allowDelete} location={location} params={params} />
```

Comment:
params not needed (probably an artifact from the branch i was using?)

Reply:
Great catch, that was remnants of the branch you were using. I'll resolve this accordingly.